src/ng/http.js

```diff
@@ -671,11 +671,11 @@ function $HttpProvider() {
 <example module="httpExample">
 <file name="index.html">
   <div ng-controller="FetchController">
-    <select ng-model="method">
+    <select ng-model="method" aria-label="HTTP method">
```

Comment:
I don't think JSONP is an HTTP method, maybe the label should be "Request method"